package.json

```diff
@@ -203,7 +203,7 @@
         "history": "^4.7.2",
         "hoist-non-react-statics": "^3.3.0",
         "immutable": "^4.0.0-rc.12",
-        "ipfs": "^0.36.4",
+        "ipfs": "^0.40.0",
```

Comment:
Using this version of ipfs, the production build runs out of memory:

![Screenshot from 2020-01-23 16-22-57](https://user-images.githubusercontent.com/1193222/72992577-a6571800-3dfc-11ea-94b3-02be6a697a3d.png)

The only way I could get this to build, is to increase node's memory limit to 4 Gb:
```
node --max-old-space-size=4096 ...
```

So, if you want us to use this version of IPFS, please add this as an runtime variable to the production build script.

Something like:
```
    ...
    "scripts": {
        ...
        "webpack:build:prod": "NODE_OPTIONS="--max-old-space-size=4096" NODE_ENV=production webpack-cli --config webpack.prod.js",
    },
    ...
```